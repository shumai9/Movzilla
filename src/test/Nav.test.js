import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter} from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from '../components/Nav';

it('renders with out failing', () =>{
  const div = document.createElement('div');
  ReactDOM.render(
      <BrowserRouter>
        <Nav />
      </BrowserRouter>, div
  );
  ReactDOM.unmountComponentAtNode(div);
});
it('renders correctly', () =>{
  const tree = renderer
    .create(
      <BrowserRouter>
        <Nav />
      </BrowserRouter>
   )
    .toJSON();
  expect(tree).toMatchSnapshot();
});