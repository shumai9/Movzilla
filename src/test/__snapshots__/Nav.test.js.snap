// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="nav-bar "
>
  <div
    className="list"
    onClick={[Function]}
  >
    <a
      aria-current={null}
      href="/home"
      onClick={[Function]}
    >
      HOME
    </a>
    <a
      aria-current={null}
      href="/about"
      onClick={[Function]}
    >
      ABOUT
    </a>
    <a
      aria-current={null}
      href="/movies"
      onClick={[Function]}
    >
      MOVIES
    </a>
    <a
      aria-current={null}
      href="/new_releases"
      onClick={[Function]}
    >
      NEW RELEASES
    </a>
    <a
      aria-current={null}
      href="/contact"
      onClick={[Function]}
    >
      CONTACT
    </a>
  </div>
</nav>
`;
